app/grandchallenge/challenges/models.py

```diff
@@ -361,6 +361,8 @@ class Challenge(ChallengeBase):
         default="",
         help_text="This email will be listed as the contact email for the challenge and will be visible to all users of Grand Challenge.",
     )
+    accumulated_compute_cost = models.FloatField(default=0, blank=True)
+    accumulated_docker_storage_cost = models.FloatField(default=0, blank=True)
```

Comment:
The name of the field should tell us if this is dollars or cents. Shouldn't we be storing things as Cents (int)?